frontend/integration-tests/views/crud.view.ts

```diff
@@ -37,15 +37,6 @@ export const rowForName = (name: string) =>
         .then((text) => text === name),
     )
     .first();
-export const rowForOperator = (name: string) =>
-  resourceRows
-    .filter((row) =>
```

Comment:
Iterating over these rows is a frequent cause of `StaleElementReferenceError`. Using `data-test-id` should make it more reliable.